### PR TITLE
Ian is no longer Kirby

### PR DESCRIPTION
**Pull request by Kyep**
🆑 Kyep
tweak: Eating as Ian (and other player-controlled corgis) is now rate-limited. 
/🆑

Previously: Ian could consume food endlessly, as fast as their player could click. This resulted in Ian being able to devor more food in a minute than their entire corgi body actually weighs. Effectively, Ian was Kirby.

Now: Ian can only take a bite of food every 30 seconds. He can still eat. He can still swipe food. But he cannot single-handedly clear out a banquet that took the chef an hour to prepare, gone in less than a minute, with little opportunity to stop him.

Effectively, this PR prevents people playing Ian from being jerks that ruin the Chef's round, whilst still allowing them to eat, self-heal, and swipe food.

### CODE REVIEW

**Comment by Citinited** [comment]
Oh yeah, that always irritated me. 👍

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -194,17 +194,22 @@
 	if(isanimal(M))
 		M.changeNext_move(CLICK_CD_MELEE)
 		if(iscorgi(M))
-			if(bitecount >= 4)
+			var/mob/living/simple_animal/pet/corgi/G = M
+			if(world.time < (G.last_eaten + 300))
+				to_chat(G, text("<span class='notice'>You are too full to try eating [src] right now.</span>"))
```
Comment (tigercat2000) [inline code/modules/food_and_drinks/food/snacks.dm]: Please never ever use the `text` proc.

Comment (Kyep) [inline code/modules/food_and_drinks/food/snacks.dm]: Changed

**Comment by scrubmcnoob** [comment]
ian you fat lard, stop eating all the food

**Comment by EldritchSigma** [comment]
No more fucking consume all ian shitters. I like it.

**Comment by taukausanake** [comment]
Awww but I LIKED being a shit and eating all the food everywhere until the chef killed me and then got arrested for station property damage. Yes, I'm one of those shitters and I don't feel bad about it. Not when I get turned into burgers afterwards

**Comment by outbools** [comment]
@taukausanake YOU SHOULD BE ASHAMED FOR YOURSELF IT'S NOT EVEN HILLARIOUS ANYMORE

**Comment by Robertha89** [comment]
Could you add like 2-3 charges to it?

File: code/modules/food_and_drinks/food/snacks.dm [hunk 2]
```diff
@@ -194,17 +194,22 @@
 	if(isanimal(M))
```
Comment (Fox-McCloud) [inline code/modules/food_and_drinks/food/snacks.dm]: Oh my gosh, why?

**Pull request merged**